nvm.sh

```diff
@@ -1848,14 +1848,25 @@ nvm_install_source() {
 
   local make
   make='make'
-  if [ "${NVM_OS}" = 'freebsd' ]; then
-    make='gmake'
-    MAKE_CXX='CXX=c++'
-  elif [ "${NVM_OS}" = 'aix' ]; then
-    make='gmake'
-  fi
-  if nvm_has "clang++" && nvm_has "clang" && nvm_version_greater_than_or_equal_to nvm_clang_version 3.5 ; then
-    nvm_echo "Clang v3.5+ detected! Use Clang as c/c++ compiler!"
+  case "${NVM_OS}" in
+    'freebsd')
+      make='gmake'
+      if [ -z "${CXX-}" ]; then
+        MAKE_CXX='CC=cc CXX=c++'
```

Comment:
Slightly different - as-is, if "CC" is set but "CXX" is unset, then "CC" will be overridden - but if the reverse is true, "CXX" won't get set.